packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -92,7 +91,7 @@ const ProofOfIdentityContainer = observer(({ height, is_from_external, onStateCh
     /**
      * Display loader while waiting for the account status and residence list to be populated
      */
-    if (is_status_loading || is_switching || isEmptyObject(account_status) || residence_list.length === 0) {
+    if (is_status_loading || is_switching || isEmptyObject(account_status)) {
```

Comment:
why do we remove the check  residence_list.length === 0 from here? 
Also then we need to change the description for this function.

Reply:
I didn't understand, which function?

Reply:
we removed the condition  **residence_list.length === 0** from the if condition. But the description still says we wait until residence list is populated.
 /**
     * Display loader while waiting for the account status and residence list to be populated
     */